components/SignUpForm/SignUpForm.tsx

```diff
@@ -45,46 +48,55 @@ export const SignUpForm = () => {
     createUserWithEmailAndPassword(auth, data.email, data.password)
       .then(async (userCredential) => {
         const user = userCredential.user;
-        await router.push(routes.SIGNIN);
+        await router.push("/");
       })
-      .catch(console.error);
+      .catch((error) => {
+        if (error.code == "auth/email-already-in-use") {
+          const errorMessage = "Email already in use";
```

Comment:
on the project you will have 100500+ errors, you will write if condition for all errors?